tools/pyboard.py

```diff
@@ -582,12 +583,18 @@ def fname_remote(src):
         return src
 
     def fname_cp_dest(src, dest):
-        src = src.rsplit("/", 1)[-1]
+        if separator == "/":
+            src = src.rsplit("/", 1)[-1]
+        else:
+            src = src.rsplit("\\", 1)[-1]
         if dest is None or dest == "":
             dest = src
         elif dest == ".":
-            dest = "./" + src
-        elif dest.endswith("/"):
+            if separator == "/":
+                dest = "./" + src
+            else:
+                dest = ".\\" + src
+        elif dest.endswith("/") or dest.endswith("\\"):
```

Comment:
On a given system, this can only be one or the other. So just `dest.endswith(os.path.sep)`